docs/kubebuilder_v0_v1_difference.md

```diff
@@ -0,0 +1,124 @@
+```
+  kubebuilder init --domain example.com
+  kubebuilder create resource --group <group> --version <version> --kind <Kind>
+  GOBIN=${PWD}/bin go install ${PWD#$GOPATH/src/}/cmd/controller-manager
+  bin/controller-manager --kubeconfig ~/.kube/config
+
+  kubectl apply -f hack/sample/<resource>.yaml
+  docker build -f Dockerfile.controller . -t <image:tag>
+  docker push <image:tag>
+  kubebuilder create config --controller-image <image:tag> --name <project-name>
+  kubectl apply -f hack/install.yaml
+```
+  Everytime the resource or controller is updated, users need to run `kubebuilder generate` to regenerate the project.
+  - kubebuilder v1 has `init`, `create api` commands and the workflow is
+  
+```
+  kubebuilder init --domain example.com --license apache2 --owner "The Kubernetes authors"
+  kubebuilder create api --group ship --version v1beta1 --kind Frigate
+  make install
+  make run
+```
+  In v1 project, there is no generate command. When the resource or controller is updated, users don't need to regenerate the project.
+  
+## scaffolding difference
+  Init a project with example.com and add a resource/api with group apps, version v1, kind Hello in both v0 and v1. The scaffolded projects are as follows.
+  - v0 project
+```
+.
+├── cmd
+│   └── controller-manager
+│       └── main.go
+├── Dockerfile.controller
+├── Gopkg.lock
+├── Gopkg.toml
+├── hack
+│   ├── boilerplate.go.txt
+│   ├── doc.go
+│   ├── imports.go
+│   └── sample
+│       └── hello.yaml
+├── pkg
+│   ├── apis
+│   │   ├── apps
+│   │   │   ├── doc.go
+│   │   │   └── v1
+│   │   └── doc.go
+│   ├── client
+│   │   ├── clientset
+│   │   │   └── versioned
+│   │   ├── informers
+│   │   │   └── externalversions
+│   │   └── listers
+│   │       └── apps
+│   ├── controller
+│   │   ├── doc.go
+│   │   └── hello
+│   │       ├── controller.go
+│   │       ├── controller_test.go
+│   │       └── hello_suite_test.go
+│   ├── doc.go
+│   └── inject
+│       ├── args
+│       │   └── args.go
+│       ├── doc.go
+│       ├── inject.go
+│       └── zz_generated.kubebuilder.go
+└── vendor
+
+```
+  - v1 project
+```
+├── cmd
+│   └── manager
+│       └── main.go
+├── config
+│   ├── crds
+│   │   └── apps_v1_hello.yaml
+│   └── manager
+│       ├── apps_rolebinding_rbac.yaml
+│       ├── apps_role_rbac.yaml
+│       └── manager.yaml
+├── Dockerfile
+├── Gopkg.toml
+├── hack
+│   └── boilerplate.go.txt
+├── Makefile
+├── pkg
+│   ├── apis
+│   │   ├── addtoscheme_apps_v1.go
+│   │   ├── apis.go
+│   │   └── apps
+│   │       ├── group.go
+│   │       └── v1
+│   │           ├── doc.go
+│   │           ├── hello_types.go
+│   │           ├── hello_types_test.go
+│   │           ├── register.go
+│   │           └── v1_suite_test.go
+│   └── controller
+│       ├── add_hello.go
+│       ├── controller.go
+│       └── hello
+│           ├── hello_controller.go
+│           ├── hello_controller_suite_test.go
+│           └── hello_controller_test.go
+├── PROJECT
+└── vendor
+```
```

Comment:
File layouts for `v0` and `v1` above makes this file verbose. I don't think it adds much of a value because it does not highlight the differences etc. So may be take it out ?

Reply:
Took it out and listed the different by bullet items.